Underscore/USConstants.h

```diff
@@ -32,6 +32,7 @@ typedef id   (^UnderscoreReduceBlock)(id memo, id obj);
 typedef void (^UnderscoreArrayIteratorBlock)(id obj);
 typedef id   (^UnderscoreArrayMapBlock)(id obj);
 typedef id   (^UnderscoreArrayZipWithBlock)(id objectFromFirstArray, id objectFromSecondArray);
+typedef void (^UnderscoreIntegerIteratorBlock)(NSInteger n);
```

Comment:
Can you make this a `NSUInteger`, obviating the need for handling the negative case?